### ISSUE TITLE
pytorch doubts

### ISSUE THREAD

**Issue by soans1994** [hmq1812/UNet-Pytorch#1]
hello author, very great work.
i have some queries since i am new to pytorch.
1. can you tell me the data dimension of image and labels while feeding it to model. how to preprocess the labels/masks? should i one hot encode? also what should be the output size of model.  i have rgb masks of many classes. i believe your mask is binary for cats and dogs.
2. also how to use pretrained weights in the encoder part of unet model. should i worry about different image resolutions at differnt stage of unet encoder. or is there a way were we just append the pretrained weights to unet model directly to the encoder output.

i will try your code. but can you please reply to my questions in simple words.  thank you
